frontend/public/components/overview/build-overview.tsx

```diff
@@ -50,17 +51,23 @@ const BuildOverviewItem: React.SFC<BuildOverviewListItemProps> = ({build}) => {
     || startTimestamp
     || creationTimestamp;
 
+  const statusTitle = (
+    <>
+      Build
+      &nbsp;
```

Comment:
Nit: I think we should prefer `{' '}` instead of `&nbsp;` HTML character entity.

Reply:
I think the &nbsp; is used here on purpose to prevent line breaks (I did not introduce this as part of my change)

Reply:
OK, understood.